app/src/layouts/DashboardLayout/MenuHeader/PremiumPlanBadge/PremiumPlanBadge.tsx

```diff
@@ -62,7 +62,10 @@ const PremiumPlanBadge = () => {
               actions.toggleActiveModal({
                 modalName: "pricingModal",
                 newValue: true,
-                newProps: { selectedPlan: null, source: "trial_badge" },
+                newProps: {
+                  selectedPlan: null,
+                  source: planStatus === "trialing" ? "trial_ongoing_badge" : "trial_expired_badge",
```

Comment:
use the constant - APP_CONSTANTS.SUBSCRIPTION_STATUS

and the create constants for the new strings.

Reply:
++